<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>osm_frame_widget</class>
 <widget class="QWidget" name="osm_frame_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>osm_frame_widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource/resource.qrc">
    <normaloff>:/ui/icons/Folder Online aqua.png</normaloff>:/ui/icons/Folder Online aqua.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget_main">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_map">
      <attribute name="icon">
       <iconset resource="resource/resource.qrc">
        <normaloff>:/ui/icons/Folder Online aqua.png</normaloff>:/ui/icons/Folder Online aqua.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Map</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="tilesviewer" name="widget_QTV_mainMap" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDockWidget" name="dockWidget_QTV_side">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>192</width>
       <height>481</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>192</width>
       <height>524287</height>
      </size>
     </property>
     <property name="windowIcon">
      <iconset resource="resource/resource.qrc">
       <normaloff>:/ui/icons/A-1 009.png</normaloff>:/ui/icons/A-1 009.png</iconset>
     </property>
     <property name="floating">
      <bool>false</bool>
     </property>
     <property name="features">
      <set>QDockWidget::AllDockWidgetFeatures</set>
     </property>
     <property name="windowTitle">
      <string>Tools</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QDial" name="dial_QTV_zoom">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>96</width>
             <height>96</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="pageStep">
            <number>4</number>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="notchTarget">
            <double>3.700000000000000</double>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLCDNumber" name="lcdNumber_QTV_level">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="smallDecimalPoint">
              <bool>false</bool>
             </property>
             <property name="digitCount">
              <number>2</number>
             </property>
             <property name="mode">
              <enum>QLCDNumber::Dec</enum>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Filled</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_QTV_saveToFile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>export</string>
             </property>
             <property name="icon">
              <iconset resource="resource/resource.qrc">
               <normaloff>:/ui/icons/3floppy_mount-1.png</normaloff>:/ui/icons/3floppy_mount-1.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="tilesviewer" name="browserView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton_QTV_active">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Activate</string>
           </property>
           <property name="icon">
            <iconset resource="resource/resource.qrc">
             <normaloff>:/ui/icons/Color Classic Green.png</normaloff>:/ui/icons/Color Classic Green.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_QTV_deactive">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Deactive</string>
           </property>
           <property name="icon">
            <iconset resource="resource/resource.qrc">
             <normaloff>:/ui/icons/Color Classic.png</normaloff>:/ui/icons/Color Classic.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_QTV_visible">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Show</string>
           </property>
           <property name="icon">
            <iconset resource="resource/resource.qrc">
             <normaloff>:/ui/icons/cn1.png</normaloff>:/ui/icons/cn1.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_QTV_hide">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Hide</string>
           </property>
           <property name="icon">
            <iconset resource="resource/resource.qrc">
             <normaloff>:/ui/icons/cn8.png</normaloff>:/ui/icons/cn8.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_QTV_layers">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_QTV_moveUp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Up</string>
           </property>
           <property name="icon">
            <iconset resource="resource/resource.qrc">
             <normaloff>:/ui/icons/up_symbol.png</normaloff>:/ui/icons/up_symbol.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_QTV_moveTop">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Top</string>
           </property>
           <property name="icon">
            <iconset resource="resource/resource.qrc">
             <normaloff>:/ui/icons/view_top_bottom.png</normaloff>:/ui/icons/view_top_bottom.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_QTV_moveDown">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Down</string>
           </property>
           <property name="icon">
            <iconset resource="resource/resource.qrc">
             <normaloff>:/ui/icons/blue - download.png</normaloff>:/ui/icons/blue - download.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_QTV_moveBtm">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Bottom</string>
           </property>
           <property name="icon">
            <iconset resource="resource/resource.qrc">
             <normaloff>:/ui/icons/view_bottom.png</normaloff>:/ui/icons/view_bottom.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>tilesviewer</class>
   <extends>QWidget</extends>
   <header>osmtiles/tilesviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dial_QTV_zoom</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_QTV_level</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
